doc/patch-author-guide.md

```diff
@@ -808,6 +808,22 @@ This can be done by replacing any usages of `static_branch_likely()`,
 `static_branch_unlikely()`, `static_key_true()`, and `static_key_false()` with
 `static_key_enabled()` in the patch file.
 
+There is an example  of using `static_key_enabled()` to replace the static key in the patch file:
+This is an static key used in the patch file:
+```
+/* skip numa counters update if numa stats is disabled */
+if (!static_branch_likely(&vm_numa_stat_key))
+    return;
+```     
+We can use `static_key_enabled()` to rewrite the `static_branch_likely` as follows::
+```
+/* skip numa counters update if numa stats is disabled */
+if (!likely(&(&vm_numa_stat_key)->key))
```

Comment:
This doesn't actually use static_key_enabled()?

Reply:
Oh! In my use case, I found that using static_branch_unlikely(&memcg_kmem_enabled_key) -> unlikely(&(&memcg_kmem_enabled_key)->key) works well. It may be a use case of fixing static key. To adjust to the patch-author-guide, I have add an new case using static_key_enabled while keeping this original use case.

Reply:
I rewrite my document and changed my test case. In this newer commit, static_key_enabled is rewrote in a macros. However, if we change the definition in header file may cause too much negative effect. So, I also provide a better solution to this situation. Thanks~